src/services/conversionRates/getConversionRatesService.js

```diff
@@ -106,21 +106,23 @@ const _getRatesCoinGecko = async (
  * @param {Number} timestamp   Timestamp for which the value should be retrieved
  * @param {Array}  ratestToGet Rates that are missing in the DB and should be retrieved
  * @param {Array}  stableCoins coins whose value equal one usd
+ * @param {String}  equivalentSymbol This field use for tokens like WBTC that has equivalent value of another currency like BTC
```

Comment:
It's not needed to pass it as a separate argument (since ratesToGet and symbol both are target), symbol or ratesToGet should be replaced with rate equivalent symbol when the function is called

Reply:
There are current arguments when this function calls:

```
 "timestamp": 1604880000000,
  "ratesToGet": [
    "ETH",
    "DAI",
    "PAN",
    "ANT",
    "BTC",
    "USDC",
    "USD",
    "AUD",
    "BRL",
    "CAD",
    "CHF",
    "CZK",
    "EUR",
    "GBP",
    "MXN",
    "THB"
  ],
  "symbol": "WBTC",
  "stableCoins": [
    "DAI",
    "SAI",
    "USDC"
  ]
```

we need `ratesToGet` and also we need `symbol` because  in addition to knowing value of a token we need the name for this line:
```
rates[symbol] = 1;
``` 
or this statement 
```
 } else {
      rates[r] = 1;
    }
```

so I think we have we should get the rate equivalent symbol in arguments or get directly from config

Reply:
I removed extra argument from _getRatesCryptocompare function